### ISSUE TITLE
Reported remediation needed for customized build failure

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-docker#37]
@CAMOBAP795 we have received a report that a customized Docker build fails like this:

<img width="728" alt="Screen Shot 2019-09-03 at 4 08 17 PM" src="https://user-images.githubusercontent.com/11865/64155282-0c9ac880-ce65-11e9-981a-25a38eab7e8c.png">

And adding the following at the beginning of the `Dockerfile` has fixed it:

> ```docker 
> RUN mkdir -p /usr/share/man/man1
> RUN apt-get update
> RUN apt-get install -y openjdk-8-jre-headless
> ``` 

Could you help confirm to see if we have this issue or not? Thanks.

**Comment by CAMOBAP795** [comment]
@ronaldtse is it possible to request minimal `Dockerfile` which help to reproduce this problem locally? (because I cannot reproduce it)

**Comment by ronaldtse** [comment]
@CAMOBAP795 I have requested it. Thanks!

**Comment by ronaldtse** [comment]
@CAMOBAP795 here's the minimal Dockerfile:

```dockerfile
FROM metanorma/metanorma
 
RUN apt-get update
RUN apt-get install -y apt-transport-https
RUN apt-get install -y curl gnupg2 software-properties-common
 
COPY mn-documents.zip
RUN unzip ./mn-documents.zip
WORKDIR mn-documents
RUN make
```

**Comment by CAMOBAP795** [comment]
Looks like we faced with https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863199 issue PR on the way

**Comment by ronaldtse** [comment]
Resolved in #38.
